prover_server/prover_server.cpp

```diff
@@ -41,50 +41,35 @@ using circuit_wrapper = libzeth::circuit_wrapper<
 namespace proto = google::protobuf;
 namespace po = boost::program_options;
 
-static void serialize_setup_to_file(
-    const typename snark::keypair &keypair,
-    boost::filesystem::path setup_path = "")
+static snark::keypair load_keypair(const boost::filesystem::path &keypair_file)
 {
-    if (setup_path.empty()) {
-        setup_path = libzeth::get_path_to_setup_directory();
-    }
-
-    const boost::filesystem::path path_vk_json = setup_path / "vk.json";
-    const boost::filesystem::path path_vk_raw = setup_path / "vk.raw";
-    const boost::filesystem::path path_pk_raw = setup_path / "pk.raw";
-
-    const typename snark::proving_key &proving_key = keypair.pk;
-    const typename snark::verification_key &verification_key = keypair.vk;
+    std::ifstream in_s(
+        keypair_file.c_str(), std::ios_base::in | std::ios_base::binary);
+    in_s.exceptions(
+        std::ios_base::eofbit | std::ios_base::badbit | std::ios_base::failbit);
+    return snark::keypair_read_bytes(in_s);
+}
 
-    // Write the verification key in json format
-    {
-        std::ofstream vk_json_s(path_vk_json.c_str());
-        snark::verification_key_write_json(verification_key, vk_json_s);
```

Comment:
Don't you write the VK in json format anymore? We need it to initialize the contracts